docs/conf.py

```diff
@@ -40,17 +40,17 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.autosummary',
+    'sphinx.ext.doctest',
     'sphinx.ext.intersphinx',
     'sphinx.ext.mathjax',
-    'sphinx.ext.doctest',
+    'sphinx.ext.napoleon',
+    'sphinx_autodoc_typehints',
```

Comment:
Don't these require additional dependencies for building?

Reply:
The latter, yes, didn't add it since I wanted to get comments first.